projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -921,9 +931,10 @@ export function checkItem(productId: string) {
     'Verifies if the given item exists within the quote cart',
     checkItem.name
   );
-  cy.get(itemsSelector + ' .cx-table-item-container .cx-info').contains(
-    productId
-  );
+  cy.get(quoteItemsComponentSelector).within(() => {
+    cy.get('.cx-info').contains(productId);
+    // cy.get('.cx-table-item-container .cx-info').contains(productId);
```

Comment:
can you check why this is commented? - either un-comment or remove

Reply:
old comment, forgot to remove it => removed!